### PR TITLE
Fix old scan tasks for use with new TMDB gem

### PR DESCRIPTION
**Pull request by XanderStrike**

### CODE REVIEW

File: config/initializers/tmdb_initializer.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+Tmdb::Api.key('a230f1c8a13699563ac819f74fb16230')
```
Comment (HParker) [inline config/initializers/tmdb_initializer.rb]: Nice. haha

**Comment by HParker** [comment]
:+1:

File: config/initializers/tmdb_initializer.rb (hunk 1 above)
Comment (XanderStrike) [inline config/initializers/tmdb_initializer.rb]: Yeah there's nothing in the terms that prohibits checking in the API key, and it's read only so they can't do bad things in my name. The worst they could do is hit their API a lot and get the key locked out.

Comment (HParker) [inline config/initializers/tmdb_initializer.rb]: Yeah, I don't have a problem with it. and it makes it much easier for me to use it. Do you plan to update the tests that used tmdb to stub it too?

Comment (XanderStrike) [inline config/initializers/tmdb_initializer.rb]: I'd like to VCR everything when I get around to it. I've used it a bunch in projects for work so I like that approach a lot.

**Pull request merged**